docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc

```diff
@@ -1,138 +1,72 @@
 [[running-on-kubernetes-standalone]]
 = Run {agent} standalone on Kubernetes
 
```

Comment:
Here as well, I am thinking the following structure:

Step 3
Step 2
Step 4 -> This step can be merged with Step2?
Step 5
Step 1